src/pages/smart-contracts/signing-transactions.md

```diff
@@ -34,7 +34,7 @@ import { openContractCall } from '@blockstack/connect';
 
 // Here's an example of options:
 const myStatus = 'hey there';
-const options = {
+const opts = {
```

Comment:
I prefer the verbose `options` vs `opts`. May I ask why you changed it? (if it's because that's the interface name in connect, perhaps we should change that to `options` in place of `opts`)

Reply:
I changed it because the sample code doesn't work.. further down in the code, `opts` is used as an input for the method call

Reply:
Yes, I would also prefer `options`, and then fixing the use of `opts` to be `options` lower in the doc.

Reply:
cool, done